src/pseudopeople/interface.py

```diff
@@ -149,7 +155,7 @@ def _generate_dataset(
                     progress_bar=False,
                 ),
                 meta=[(c.name, c.dtype_name) for c in dataset_schema.columns],
-            )
+            )  # pd.DataFrame
```

Comment:
What does this do?

Reply:
Nothing, it's just a note to the user that it's a pd.DataFrame. Type hinting as such threw a different type of mypy error